setup.py

```diff
@@ -56,7 +59,7 @@
 }
 
 requirements = [
-    "torch==2.3.1",
+    f"torch=={TORCH_VERSION}",
```

Comment:
I'd really prefer to see this get unlocked with a `>-` instead of `==`.

Reply:
It's acceptable. I'll make the changes here and in the AWQ package too once I've synced it with upstream.

Reply:
https://github.com/casper-hansen/AutoAWQ_kernels/pull/29/commits/c3a922fdb070240fa5e602155f130e573a38120a should do just fine.